.github/workflows/ibis-backends.yml

```diff
@@ -329,6 +349,25 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: delete header rows on ubuntu
+        if: matrix.os == 'ubuntu-latest' && matrix.backend.name == 'flink'
+        run: |
+          python -c "exec(\"TEST_FILES = ['functional_alltypes', 'diamonds', 'batting', 'awards_players']\nfor file in TEST_FILES:\n    with open(f'ci/ibis-testing-data/csv/{file}.csv', 'r+') as f:\n        f.readline()\n        data=f.read()\n        f.seek(0)\n        f.write(data)\n        f.truncate()\n\")"
+
+      - name: delete header rows on windows
+        if: matrix.os == 'windows-latest' && matrix.backend.name == 'flink'
+        run: |
+          python -c @"
+          TEST_FILES = ['functional_alltypes', 'diamonds', 'batting', 'awards_players']
+          for file in TEST_FILES:
+              with open(f'ci/ibis-testing-data/csv/{file}.csv', 'r+') as f:
+                  f.readline()
+                  data=f.read()
+                  f.seek(0)
+                  f.write(data)
+                  f.truncate()
+          "@
+
```

Comment:
I'll remove this once CI finishes once; will cherry-pick it over from the other PR.